src/deployed.json

```
@@ -647,6 +647,15 @@
                 "iconAddress": "0xf650C3d88D12dB855b8bf7D11Be6C55A4e07dCC9",
                 "precision": 2,
                 "chartColor": "#557064"
+            },
+            {
+                "symbol": "RSR",
+                "address": "0x8762db106b2c2a0bccb3a80d1ed41273552616e8",
```

The address is wrong here, it should be checksumed
```suggestion
                "address": "0x8762db106B2c2A0bccB3A80d1Ed41273552616E8",
```
Same apply for the `iconAddress`. Make sure the image works, example: https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x8762db106B2c2A0bccB3A80d1Ed41273552616E8/logo.png